config/log.conf

```diff
@@ -3,5 +3,21 @@ log_format main_default escape=default
   '"$request" $body_bytes_sent "$http_referer" '
   '"$http_user_agent" "$http_x_forwarded_for" $proxy_x_request_id';
 
+log_format main_json escape=json
+  '{'
+    '"body_bytes_sent":"$body_bytes_sent",'
+    '"http_referrer":"$http_referer",'
+    '"http_user_agent":"$http_user_agent",'
+    '"http_x_forwarded_for":"$http_x_forwarded_for",'
+    '"proxy_x_request_id":"$proxy_x_request_id",'
+    '"remote_addr":"$remote_addr",'
+    '"remote_user":"$remote_user",'
+    '"request":"$request",'
+    '"request_length":"$request_length",'
+    '"request_time":"$request_time",'
+    '"status": "$status",'
+    '"time_local":"$time_local"'
```

Comment:
To note, the last attribute has no `,`. Also; two questions:

1. Is there anything we'd like to add?
2. Is there anything we'd like to rename?

Reply:
I've added `$host`, `$proxy_x_forwarded_port`, `$proxy_x_forwarded_proto` and `$proxy_x_forwarded_ssl` in https://github.com/Intellection/docker-nginx-proxy/commit/05abb6724e8cd8bf5ff3fa7a0f2d0ca533fd6ba4.

Reply:
I think it's probably okay for now.